cmd/test-server/zerotrace.go

```diff
@@ -62,10 +62,11 @@ func newZeroTrace(iface string, conn net.Conn) *zeroTrace {
 	clientPort, _ := strconv.Atoi(clPort)
 
 	return &zeroTrace{
-		Iface:      iface,
-		Conn:       conn,
-		ClientIP:   clientIP,
-		ClientPort: clientPort,
+		Iface:        iface,
+		Conn:         conn,
+		ClientIP:     clientIP,
+		ClientPort:   clientPort,
+		SentPktsIPId: make(map[int][]SentPacketData),
```

Comment:
I didn't realize we could initialize a map straightaway here. Instead I had added a line to initialize it in `recvPackets()` where `processTCPpkt()` is called to write into it. Thanks for highlighting this.